usage/rest-client/pom.xml

```diff
@@ -114,14 +114,14 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-core</artifactId>
             <version>${project.version}</version>
+            <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-core</artifactId>
             <version>${project.version}</version>
-            <classifier>tests</classifier>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
was brought in to use `HttpTool.isStatusCodeHealthy()`.  It wouldn't
compile without the `compile` scope.

On 19 June 2015 at 13:15, Alex Heneveld <notifications@github.com> wrote:

> In usage/rest-client/pom.xml
> <https://github.com/apache/incubator-brooklyn/pull/690#discussion_r32823340>
> :
>
> >              <scope>test</scope>
> >          </dependency>
> >          <dependency>
> >              <groupId>org.apache.brooklyn</groupId>
> >              <artifactId>brooklyn-core</artifactId>
> >              <version>${project.version}</version>
> > -            <classifier>tests</classifier>
> > -            <scope>test</scope>
> > +            <scope>compile</scope>
>
> why do we need to add <scope>compile</scope>?
>
> previously both dependencies, regular and <classifier>tests</classifier>
> were only brought in for test scope I think which lowers the footprint
> for someone using the REST client. also the ordering of the regular
> classifier first is cleaner i think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-brooklyn/pull/690/files#r32823340>.
>

-- 
Cloudsoft Corporation Limited, Registered in Scotland No: SC349230. 
 Registered Office: 13 Dryden Place, Edinburgh, EH9 1RP
 
This e-mail message is confidential and for use by the addressee only. If 
the message is received by anyone other than the addressee, please return 
the message to the sender by replying to it and then delete the message 
from your computer. Internet e-mails are not necessarily secure. Cloudsoft 
Corporation Limited does not accept responsibility for changes made to this 
message after it was sent.

Whilst all reasonable care has been taken to avoid the transmission of 
viruses, it is the responsibility of the recipient to ensure that the 
onward transmission, opening or use of this message and any attachments 
will not adversely affect its systems or data. No responsibility is 
accepted by Cloudsoft Corporation Limited in this regard and the recipient 
should carry out such virus and other checks as it considers appropriate.